scripts/objectiveValidation.js

```diff
@@ -10,7 +10,7 @@ async function execGitWithArgs(helper, argString, execOptions = {}) {
     );
   }
 
-  return exec(`${TQ_GIT_EXE} ${argString}`, execOptions);
```

Comment:
Let's add a check for leading and trailing `"` and `'` from the TQ_GIT_EXE arg.

We should throw a new error if these exist with a message:
> TwilioQuest no longer requires you to surround your git executable path with quotation marks if it contains spaces.

Let's also put a comment to say that we advised users to add quotations marks to get past this validator after the TQ 3.10.0 release introduced the bug.